build_service/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+
+REMOTE_URL=https://github.com/secure-device-onboard/epid-verification-service.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ]; then
+    echo "Building $REMOTE_URL : $REMOTE_BRANCH"
+    cd /tmp/
```

Comment:
TMPDIR may be set?  If so, should you prefer that directory?

Reply:
The same has been used in other SDO components' out of box build, so `/tmp/` directory is used here for ensuring uniformity across various codebase